test/accounts.js

```diff
@@ -21,17 +21,15 @@ describe("Accounts", async() => {
     const { accounts, web3 } = await initializeTestProvider(options);
 
     accounts.forEach(async(account) => {
-      try {
-        await web3.eth.sendTransaction({
+      assert.rejects(
```

Comment:
You need to await calls to `assert.rejects`.

Reply:
Turns out `assert.rejects` was added in v10.0 and breaks when tested against `carbon`. The `try/catch` pattern is the cleanest approach that passes Travis CI across the board.

Reply:
Node docs show that it was added in V8.13.0: https://nodejs.org/docs/latest-v8.x/api/assert.html#assert_assert_rejects_block_error_message

Reply:
The second arg must be one of `<RegExp> | <Function> | <Object>`. You could replace the `error.message` string comparison check with something like `/signer account should be locked/` (this regexp is the same as calling `err.message.toLowerCase().indexOf("signer account should be locked") !== -1`)